### PR TITLE
added issue and pr templates

### PR DESCRIPTION
**Pull request by Sloth-Panda**
issue: #23

### CODE REVIEW

**Comment by Sloth-Panda** [comment]
@kritikaparmar-programmer

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
```
Comment (kritikaparmar-programmer) [inline .github/config.yml:1]: Remove this config file as it is already added by someone in a pull request then it can create conflicts.

Comment (Sloth-Panda) [reply .github/config.yml:1]: @kritikaparmar-programmer thats just  a comment

Comment (Sloth-Panda) [reply .github/config.yml]: I have updated it anyways ☺

**Comment by Sloth-Panda** [comment]
@kritikaparmar-programmer please review this pr and also  please add gssoc21 and level tags to both issue and pr

**Comment by Sloth-Panda** [comment]
@utkarsh0702 can you review it

**Comment by utkarsh0702** [comment]
@Sloth-Panda I will discuss with the project admin and other mentors and will then merge the pr.

**Comment by kritikaparmar-programmer** [comment]
Hey, @Sloth-Panda I was asking you to remove `.github/config.yml ` file. 
remove this file then I'll merge the pr

**Comment by Sloth-Panda** [comment]
@kritikaparmar-programmer hey, I have removed the file you can check

**Review by kritikaparmar-programmer — APPROVED** [review]
GREAT 🚀

**Pull request merged**